vocabulary/favicon-html.md

```diff
@@ -0,0 +1,21 @@
+# favicon
+
+## A favicon is a small image displayed next to the page title in the browser tab
+
+## It is a small image, so it should be a simple image with high contrast
+
+## To add a favicon to your website, either save your favicon image to the root directory of your webserver, or create a folder in the root directory called images, and save your favicon image in this folder. A common name for a favicon image is "favicon.ico" Next, add a `link` element to your "index.html" file, after the `title` element, like this
+
+<!DOCTYPE html>
+        <html>
```

Comment:
The html element is duplicated, which is invalid in HTML. 
Also consider indenting your code properly to make it more readable

Reply:
Yes that was not meant to be duplicated.
Question: When doing html in VS code is there a way of formatting before pushing so I can be sure it is readable?

Reply:
The VS Code has two explicit format actions: 
**Format Document** (Ctrl+Shift+I)
**Format Selection** (Ctrl+K Ctrl+F)

ALT+SHIFT+F works too if you select a formatter (the default works well, but you can also use ESLint for example)

Reply:
@ThibautHumblet Awesome thanks for the tips ;)